app/actions/goals-list.js

```diff
@@ -0,0 +1,6 @@
+import { createAction } from 'redux-actions';
```

Comment:
Why are you using `redux-actions`?

The other way of creating this function is
`const stepAddGoal = () => ({ type: 'STEP_ADD_GOAL' })` 

create action looks nicer so am happy with that but wanted to make sure you knew how to do it vanilla.